db/install.php

```diff
@@ -0,0 +1,47 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Install script for Components for Learning (Elements)
+ *
+ * Documentation: {@link https://moodledev.io/docs/guides/upgrade}
+ *
+ * @package    tiny_elements
+ * @copyright  2024 ISB Bayern
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+use tiny_elements\importer;
+
+/**
+ * Executed on installation of Components for Learning (Elements)
+ *
+ * @return bool
+ */
+function xmldb_tiny_elements_install() {
+    // This is for debugging purposes only and should be removed in production.
+    return true;
```

Comment:
can be removed again. Also the cli script can be removed again.

Reply:
I'd keep the CLI script - could be useful for reinstalling base.zip templates without UI.